pkg/controller/controller_instance.go

```diff
@@ -583,9 +621,27 @@ func (c *controller) reconcileServiceInstance(instance *v1alpha1.ServiceInstance
 			v1alpha1.ConditionFalse,
 			errorErrorCallingProvisionReason,
 			"Provision call failed and will be retried: "+s)
```

Comment:
There is ambiguity here as to whether the instance reconciliation will be retried or not. The ready condition says it will, but the failed message says it won't.